exercises/book-store/canonical-data.json

```diff
@@ -85,6 +85,15 @@
          },
          "expected": 5120
         },
+        {
+         "property": "total",
+         "description": "Two groups of four is cheaper than group of five plus group of three, regardless of order",
```

Comment:
I would suggest to re-use the terminology of the last test case:

"Two groups of four are cheaper than two groups of five and three"

Do we need the "regardless of order" addition? I feel this is better suited for the comment, in which it already is explained.

Reply:
> Do we need the "regardless of order" addition? I feel this is better suited for the comment, in which it already is explained.

I agree.  `regardless of order` implies that we know this to be true.  There are no tests for every possible combination to confirm this accusation to be true.

Reply:
Scaled back the description per your comments.